Kudu.Core/Deployment/Generator/GeneratorSiteBuilder.cs

```diff
@@ -20,12 +20,6 @@ protected GeneratorSiteBuilder(IEnvironment environment, IDeploymentSettingsMana
         {
         }
 
-        // TODO: Do we still need this
-        /*protected string GetPropertyString()
-        {
```

Comment:
this is a piece of unreferenced code that was causing an error during upgrade. I removed it, however, it can be left in case it's needed later. Totally up to you.